docs/lua-modding/lua-intro.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+Lets set some ground rules so you don't complain about us wasting your time:
+- This is a _**Lua-focused**_ tutorial. There may be portions on BP interoperability (eventually), but for the most part this will be pure, untainted Lua 😎 Some concepts will definitely transfer over though.
+- We are **not** going to cover Lua basics, like syntax or structures etc, in the main tutorials. We have a wonderful [Lua Basics](https://pwmodding.wiki/docs/Lua-Modding/LuaBasics) page for that. Please get yourself acquainted with Lua beforehand.
```

Comment:
maybe rephrase this to tell people to go for lua basics there, saying we are not going to cover it and then providing a link where we do is kinda contradictory